### PR TITLE
feat(dashboard): Add overview dashboard 

### PR DESCRIPTION
**Pull request by billyvg**
image

Depends on
* ~~https://github.com/getsentry/sentry/pull/11434~~
* ~~https://github.com/getsentry/sentry/pull/11408~~
* https://github.com/getsentry/sentry/pull/11428
* ~~https://github.com/getsentry/sentry/pull/11425~~

### CODE REVIEW

**Comment by billyvg** [comment]
Please review the last commit

File: src/sentry/static/sentry/app/views/organizationDashboard/data/queries/anonymousUsersAffected.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const anonymousUsersAffectedQuery = {
+  fields: [],
+  conditions: [['user.email', 'IS NULL', null]],
+  aggregations: [['count()', null, 'Anonymous Users']],
+  limit: 1000,
+
+  orderby: '-time',
+  groupby: ['time'],
+  rollup: 86400,
+};
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationDashboard/data/queries/anonymousUsersAffected.jsx]: Is this the shape of the data we're thinking of storing in the dashboard widget tables?

Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationDashboard/data/queries/anonymousUsersAffected.jsx]: Yeah, this is the same shape as discover queries

File: src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    // Reset query builder
+    this.resetQueries();
+
+    // Fetch
+    const promises = this.queryBuilders.map(builder => builder.fetch());
+    const systemPromises = this.systemQueryBuilders.map(
+      builder => (builder ? builder.fetch() : null)
+    );
+    let results = await Promise.all(promises);
+    let systemResults = await Promise.all(systemPromises);
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx]: I think I don't quite understand what results vs systemResults is supposed to be here

File: src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx]: nit: filename should be `discoverQuery` vs `discoveryQuery`

File: src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx]: It's currently unused, but the idea is that it's a generated query based on the user's query. Example being we want the same query but with a different time period (from 15 days ago to 8 days ago)

File: src/sentry/static/sentry/app/views/organizationDashboard/overviewDashboard/overviewDashboard.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import Dashboard from 'app/views/organizationDashboard/dashboard';
+import overviewDashboard from 'app/views/organizationDashboard/data/dashboards/overviewDashboard';
+
+class OverviewDashboard extends React.Component {
+  render() {
+    return <Dashboard {...overviewDashboard} />;
+  }
+}
+export default OverviewDashboard;
+export {OverviewDashboard};
```
[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationDashboard/overviewDashboard/overviewDashboard.jsx]: Removed https://github.com/getsentry/sentry/pull/11416/commits/fab1b9e06f128bef604014cb6366939e5c3ca1ff

File: src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx (hunk 3 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx]: thanks, fixed https://github.com/getsentry/sentry/pull/11416/commits/73039827f867a437c61918a36c7d9190c6cd0961

**Comment by billyvg** [comment]
> This looks generally good to me - the only thing I'd ask is if we can pull out the "system queries" stuff from this PR since it's a bit confusing and isn't being used anywhere

Removed https://github.com/getsentry/sentry/pull/11494

File: src/sentry/static/sentry/app/views/organizationDashboard/discoverQuery.jsx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    selection: SentryTypes.GlobalSelection,
+    queries: PropTypes.arrayOf(SentryTypes.DiscoverQuery),
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      results: null,
+      systemResults: null,
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationDashboard/discoverQuery.jsx]: This line isn't needed anymore

**Review by lynnagara — APPROVED** [review]
💯

**Pull request merged**